benchmarks/benchmarks/stats.py

```diff
@@ -3,15 +3,17 @@
 import warnings
 
 import numpy as np
+from numpy.random import rand
 
 try:
     from scipy.stats import anderson_ksamp
+    from scipy.stats import fisher_exact
+    import scipy.stats as stats
```

Comment:
I would change the 3 imports above to ``from scipy import stats`` and adapt the code below to that.